apps/src/applab/designElements/elementUtils.js

```diff
@@ -120,4 +120,13 @@ module.exports.isIdAvailable = function(newId, originalId) {
   }
 
   return true;
+};
+
+module.exports.isContentEditable = function (element) {
+  return element && element.tagName.toUpperCase() === 'DIV' && element.contentEditable;
```

Comment:
We can't only use isTextArea without changing the approach, because we only want to use the synthetic change event handler `if ((elementUtils.isContentEditable(domElement) || elementUtils.isTextInput(domElement))`. I think a better approach is to mark the "text area" with a data attribute or class so that we know what it is regardless of whether it is currently readonly. How does that sound?